plugins/azure-devops/src/alpha.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from './alpha/index';
+export { default } from './alpha/index';
```

Comment:
```suggestion
export * from './alpha';
export { default } from './alpha';
```

if you had these here to circumvent a circular import, you should probably instead rename that folder.

Or structure it differently in some other way. Note how this one for example has a dedicated plugin.ts and only does the `as default` dance in the root index file, not at deeper folder layers https://github.com/backstage/backstage/blob/2cfd6f55e31b96b75fcd7c45bc6445e5644a0e5e/plugins/badges-backend/src/index.ts#L28

Reply:
Honestly I just copied what I saw here: https://github.com/backstage/backstage/blob/56d3373c8262e947747b7dfe399d9a5870ff775a/plugins/catalog/src/alpha.ts#L17-L18. I wasn't sure how to setup alpha exports.

Reply:
@freben, any thoughts on this? Happy to make changes just honestly not sure how.